spring/boot2-webflux-autoconfigure/src/test/java/com/linecorp/armeria/spring/web/reactive/ArmeriaServerHttpResponseTest.java

```diff
@@ -171,6 +172,30 @@ void returnHeadersAndBody() throws Exception {
         await().until(() -> httpResponse.whenComplete().isDone());
     }
 
+    @Test
+    void ignoreCancelledSubscriptionException() throws Exception {
+        final ServiceRequestContext ctx = ServiceRequestContext.of(HttpRequest.of(HttpMethod.HEAD, "/"));
+
+        final CompletableFuture<HttpResponse> future = new CompletableFuture<>();
+        final ArmeriaServerHttpResponse response = response(ctx, future);
+
+        response.setStatusCode(HttpStatus.OK);
+        response.getHeaders().add("Armeria", "awesome");
+        assertThat(future.isDone()).isFalse();
+
+        StepVerifier.create(Mono.defer(response::setComplete))
+                    .then(() -> {
+                        try {
+                            // throw CancelledSubscriptionException as HttpResponseSubscriber
+                            // cancels subscription for HTTP HEAD
+                            final HttpResponse httpResponse = future.get();
+                            httpResponse.whenComplete()
+                                        .completeExceptionally(CancelledSubscriptionException.get());
+                        } catch (Throwable ignored) { }
```

Comment:
nit: extra space in `{ }`.

Reply:
when I make it as `{}` Checkstype fails with 
```
 WhitespaceAround: '{' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
```

let me just make it as 
```
} catch (Throwable ignored) {
}
```

Reply:
Oops, didn't know about it. 😅